src/Bicep.Core.Samples/Files/InvalidExpressions_LF/main.bicep

```diff
@@ -56,6 +56,7 @@ var div = {
 // additive
 var add = null + 's'
 var sub = true - false
+var add = 'bad' + 'str'
```

Comment:
Totally up to you, but I generally also like to add specific scenario tests (`ScenarioTests.cs`) when I'm adding new features, as I feel like they're less likely to get regressed.

Reply:
Done!